scifin/timeseries/timeseries.py

```diff
@@ -654,25 +676,25 @@ def annualized_return(self, start=None, end=None):
             raise ValueError('Annualized return could not be evaluated.')
     
     
-    def risk_ratio(self, start=None, end=None):
+    def risk_ratio(self, start=None, end=None, verbose=True):
         """
         Returns the risk ratio, i.e. the ratio of annualized return
         over annualized volatility.
         """
 
         ann_return = self.annualized_return(start, end)
-        ann_volatility = self.annualized_vol(start, end)
+        ann_volatility = self.annualized_vol(start, end, verbose=verbose)
         
         return ann_return / ann_volatility
 
     
-    def annualized_Sharpe_ratio(self, risk_free_rate=0, start=None, end=None):
+    def annualized_Sharpe_ratio(self, risk_free_rate=0, start=None, end=None, verbose=True):
```

Comment:
Why don't you add a verbose attribute to the class rather than polluting all methods with this argument?
Of course this is also a matter of taste

Reply:
I did not think about it. I must say it would make this attribute a bit different from others and we may want some functions to be verbose=True by default while others having verbose=False. If we make verbose into an attribute, we'll have to change the attribute each time we want to stop/start displaying info. Otoh it would simplify many methods, so I have mixed feelings about this change.

Let's not do this change for the moment (and rather remove start and end arguments).